### ISSUE TITLE
Option to render proper scrollbar

### ISSUE THREAD

**Issue by tinecek350** [microsoft/vscode#207762]
he current VS Code scrollbars (both vertical and horizontal) are OK, but I would prefer:

The scrollbars not to disappear when the mouse stops moving
The scrollbars to be a bit thicker, thus easier to click
An up and a down arrow at the top and bottom (or left and right of the horizontal one) to scroll one line per click
The scrollbar to jump "towards" where you click on empty space either side of the scroll handle, instead of the handle immediately jumping to where the mouse cursor is
In short, they should be like the Visual Studio 2015 scrollbars. Maybe a setting could be introduced to render this more "traditional" type of scrollbar.

**Comment by gjsjohnmurray** [comment]
Please see https://code.visualstudio.com/updates/v1_58#_editor-scroll-bar-customization and https://github.com/microsoft/vscode/issues/66000
